src/Referent/ReferentMessageNotifier.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace AppBundle\Referent;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\ReferentManagedUsersMessage;
+use AppBundle\Producer\ReferentMessageDispatcherProducerInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use Ramsey\Uuid\Uuid;
+
+class ReferentMessageNotifier
+{
+    private $manager;
+    private $producer;
+
+    public function __construct(
+        ObjectManager $manager,
+        ReferentMessageDispatcherProducerInterface $producer
+    ) {
+        $this->manager = $manager;
+        $this->producer = $producer;
+    }
+
+    public function createMessage(Adherent $referent, ManagedUsersFilter $filter)
```

Comment:
This is not the responsibility of this class, you should add a static method in `ReferentMessage` instead.